app/Console/Kernel.php

```diff
@@ -25,7 +26,7 @@ protected function schedule(Schedule $schedule): void
         $schedule->command(DispatchQueueCheckJobsCommand::class)->everyMinute();
 
         $schedule->command(PruneCommand::class, [
-            '--model' => [Audit::class, AssistantChatMessageLog::class],
+            '--model' => [Audit::class, AssistantChatMessageLog::class, FailedImportRow::class],
```

Comment:
Do we need to add a task to the backlog to make the pruning settings configurable?

Reply:
Yeah, good idea to add to the backlog

Reply:
Created the ticket for this: https://engage.canyongbs.com/workgroups/group/3/tasks/task/view/451/